client/landing/stepper/declarative-flow/internals/steps-repository/design-carousel/style.scss

```diff
@@ -1,5 +1,123 @@
+@import "../style";
+
 .designCarousel.step-container .step-container__header .formatted-header {
 	.formatted-header__subtitle {
 		text-align: center;
 	}
 }
+
+.design-carousel__select {
+	// Hide Gridicon Heart icon by default
+	svg {
+		display: none;
+	}
+}
+
+.link-in-bio.patterns,
+.link-in-bio-tld.patterns {
```

Comment:
Should these be just "mobile patterns" instead of scoping them to a single flow? Makes it less cumbersome to re-use for other flows in future.

Reply:
Yes, totes fine for a follow-up. Thank you!